aiogram_dialog/context/intent_filter.py

```diff
@@ -117,16 +144,12 @@ async def on_pre_process_callback_query(self, event: CallbackQuery, data: dict):
             if stack.last_intent_id() != intent_id:
                 logger.warning(f"Outdated intent id ({intent_id}) for stack ({stack.id})")
                 raise CancelHandler()
+            event.Config.allow_mutation = True  # ToDo
```

Comment:
huh

Reply:
oh, missed that.

How to do it better?
Redo the handling of callback.data in filters and ignore prefix?